raster/r.texture/testsuite/test_texture.py

```diff
@@ -23,60 +24,192 @@ def setUpClass(cls):
     @classmethod
     def tearDownClass(cls):
         cls.del_temp_region()
-        cls.runModule("g.remove", flags="f", type="raster", name="asm_ASM")
-        cls.runModule("g.remove", flags="f", type="raster", name="corr_Corr")
-        cls.runModule("g.remove", flags="f", type="raster", name="sa_SA")
-        cls.runModule("g.remove", flags="f", type="raster", name="var_Var")
-        cls.runModule("g.remove", flags="f", type="raster", name="idm_IDM")
+        maps = [
+            "asm_ASM",
+            "contrast_Contr",
+            "corr_Corr",
+            "var_Var",
+            "idm_IDM",
+            "sa_SA",
+            "sv_SV",
+            "se_SE",
+            "entr_Entr",
+            "dv_DV",
+            "de_DE",
+            "moc1_MOC-1",
+            "moc2_MOC-2",
+        ]
+        maps_string = ",".join(maps)
+        cls.runModule("g.remove", flags="f", type="raster", name=maps_string)
```

Comment:
The underlying functions should deal with a list and do the join for the command line interface automatically:

```suggestion
        cls.runModule("g.remove", flags="f", type="raster", name=maps)
```